### ISSUE TITLE
Personal Goals

### ISSUE THREAD

**Issue by henryhilton15** [dartmouth-cs98/Quantum-Tetris#168]
# Name | Personal Goals
*what does success for this class mean to you?*

* coming out with a sleek, impressive product that we are excited to post around the internet and get some real world traction
* being able to explain complex quantum concepts in layman's terms
* gaining confidence in my web development and graphic design skills

## Challenges
*what are some challenges you foresee?*

* getting too ambitious about adding new features before we finalize the polishing and design that will make a respectable real world product that is not hacky or "student project" seeming

**Comment by timofei7** [comment]
i like the focus on polish and the frontend work you've done.  and there are fun challenges ahead!
